flow/Makefile.py

```diff
@@ -0,0 +1,98 @@
+import argparse
```

Comment:
Some explanatory comments would be helpful to make clear how this is intended to be used.  Perhaps a fuller README

Reply:
Good point, thanks. I'll add some comments to explain the argument parsing and expand the README in flow/scripts/sc.